data/schema-config.yaml

```diff
@@ -934,6 +934,29 @@ regulatory element to gene expression association to biosample to donor:
     source: str
     source_url: str
 
+regulatory element to biosample:
+  is_a: ontology class
+  represented_as: edge
+  label_in_input: regulatory_region_biosample
+  label_as_edge: REGULATORY_REGION_BIOSAMPLE
+  db_collection_name: regulatory_regions_biosamples
+  db_collection_per_chromosome: false
+  relationship:
+    from: regulatory region
+    to: ontology term
+  accessible_via:
+    name: regulatory_regions_biosamples
+    return: _id, source, source_url
```

Comment:
You defined a lot properties that are not returned here, for example element_name, strand and activity_score. I think we should return those properties.

Reply:
Yes I will add them.